Add instructions for using the shell to move files (#1)
UCL/myriad-stardist-pipeline: README.md

    └── ...
```

**On your local machine**, you can arrange your files into the above directory structure and zip/compress them
as normal.

You can then [run commands in the shell](https://swcarpentry.github.io/shell-novice/index.html#open-a-new-shell)

> to transfer them to your Myriad `Scratch` space using `scp`:

Reviewer: ```suggestion
to transfer them to the `myriad-stardist-pipeline` directory in your Myriad home directory using `scp`:
```

```bash

> scp models.zip ucaXXXX@myriad.rc.ucl.ac.uk:~/Scratch/
> scp images.zip ucaXXXX@myriad.rc.ucl.ac.uk:~/Scratch/

Reviewer: ```suggestion
scp models.zip ucaXXXX@myriad.rc.ucl.ac.uk:~/myriad-stardist-pipeline/
scp images.zip ucaXXXX@myriad.rc.ucl.ac.uk:~/myriad-stardist-pipeline/
```

```

> N.B. `ucaXXXX` is your UCL username and this command is similar to `ssh` command you used to connect to Myriad.

Reviewer: ```suggestion
> [!NOTE]
> `ucaXXXX` is your UCL username and this command is similar to `ssh` command you used to connect to Myriad.
```

> **On myriad**, you can check the contents of your `Scratch` space and any other folder by running the

Reviewer: ```suggestion
**On Myriad**, you can check the contents of your `myriad-stardist-pipeline` directory and any other folder by running the
```

[ls command](https://swcarpentry.github.io/shell-novice/02-filedir.html):

```bash

> cd ~/Scratch

Reviewer: ```suggestion
cd ~/myriad-stardist-pipeline
```

ls
```

> You should see the two .zip folders you just transferred.

Reviewer: ```suggestion
You should see the two `.zip` files you just transferred.
```

> N.B. You can also check which folder/directory you're currently in by running the `pwd` command.

Reviewer: ```suggestion
> [!TIP]
> You can also check which folder/directory you're currently in by running the `pwd` command.
```

> 
> Next, you need to move the zipped files to the `myriad-stardist-pipeline` directory you created earlier
> using the [`mv` command](https://swcarpentry.github.io/shell-novice/03-create.html#moving-files-and-directories
> ):
> 
> ```bash
> cd ~
> mv ~/Scratch/models.zip ~/myriad-stardist-pipeline/
> mv ~/Scratch/images.zip ~/myriad-stardist-pipeline/

Reviewer: ```suggestion
```

With suggested changes above we can remove this separate `mv` step

```

Unzip the files in your `myriad-stardist-pipeline` directory using the `unzip` command:

```bash

> cd ~/myriad-stardist-pipeline

Reviewer: ```suggestion
```

Reviewer: With changes above should already be in the `myriad-stardist-pipeline` directory at this point

unzip models.zip
unzip images.zip
```

You can check the contents of the `models` and `images` directories to confirm that the files have been unzipped correctly:

```bash
cd models
ls
cd ../images
ls
```

Submit job script by running

```bash

> cd ~

Reviewer: ```suggestion
cd ~/myriad-startdist-pipeline
```

Reviewer: The job script is in `myriad-stardist-pipeline` directory rather than home.
